onnxruntime/contrib_ops/webgpu/quantization/dp4a_matmul_nbits.cc

```diff
@@ -39,6 +39,18 @@ constexpr std::string_view commonFunctions = R"ADDNL_FN(
         }
   )ADDNL_FN";
 
+constexpr std::string_view AlignWithZeroPoint = R"ADDNL_FN(
+        fn AlignWithZeroPoint(in: vec4<u32>) -> vec4<u32>
+        {
+            var out = vec4<u32>(0);
+            out[0] = pack4xI8(vec4<i32>(unpack4xU8(in[0])) - vec4<i32>(128));
```

Comment:
pack4xI8(vec4<i32>(unpack4xU8(in[0]) - vec4<i32>(128)));
otherwise uint 129 will immediately become -127 - 128.